Docs 2018.2

josephsl opened the pull request:
Hi,

As usual, this is a regular fixes,typo fixes/issue numbering and so on for NvDA 2018.2. In particular, user guide and what's new has been edited to add two new languages, some issues were renumbered, linguistic fixes and so on.

Thanks.

josephsl committed:
User guide: reorder languages list.

Reorder languages list to follow sorted order based on languages. When appropriate, list locales in braces.

josephsl committed:
User guide and what's new: add Mongolian and Swiss German to list of new languages.

josephsl committed:
Merge branch 'master' into docs18.2

# Conflicts:
#	user_docs/en/changes.t2t

josephsl committed:
What's new: linguistics and issue number fixes.

josephsl committed:
Merge branch 'master' into docs18.2

leonardder commented:
Note that #7302 also contains some major rewrites of the user guide. May be you could read those changes and file a pr against the @babbagecom repo to have them fixed as part of #7302? That is, if you find any mistakes

josephsl committed:
Merge branch 'master' into docs18.2

josephsl committed:
What's new: linguistic fixes.

josephsl committed:
User guide: linguistics, settings category descriptions.

Added descriptions for categories for ones that does not feature it (for consistency purposes). Also, did some linguistic fixes.

josephsl committed:
Merge branch 'master' into docs18.2

# Conflicts:
#	user_docs/en/changes.t2t

josephsl commented:
Hi,

All done, including editing #7302 docs. I recommend a review and merge on the day 2018.2 strings are frozen.

Thanks.
